lib/certificate-stack.ts

```diff
@@ -42,7 +42,7 @@ export class CertificateStack extends cdk.NestedStack {
                             "curl --retry 3 --retry-delay 5 https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip",
                             "unzip awscliv2.zip",
                             "./aws/install",
-                            "pip3 install -i https://opentuna.cn/pypi/web/simple certbot==1.9.0 certbot-dns-route53==1.9.0 cryptography==3.1.1",
+                            "pip3 install -i https://opentuna.cn/pypi/web/simple certbot==1.11.0 certbot-dns-route53==1.11.0 cryptography==3.3.1",
```

Comment:
What’s the purpose of pinning `cryptography`?

Had a deep dive into `certbot-dns-route53`, which depends on  both `pyopenssl` and `cryptography` with opening range. The previous error was caused by `cryptography` pin as `3.1.1`, however the latest `pyopenssl` requires the minimum `cryptography` newer than `3.1.1`, it causes the dependencies conflict.

If there is no particular reason for pinning `cryptography` to a certain version, the fix should remove the certain `cryptography` safely.